README.md

```diff
@@ -17,22 +17,26 @@ Allows you to run [Apex Static Analysis](http://pmd.sourceforge.net/snapshot/pmd
 ## Installation
 
 1. Must have `JDK >=1.4` installed and in path
-2. Download a [PMD "bin" release](https://sourceforge.net/projects/pmd/files/pmd/) (`>= 5.6`)
+2. Download a [PMD "bin" release](https://sourceforge.net/projects/pmd/files/pmd/) (`>= 6.0`)
 3. unzip to location of choice
 4. In VScode, Open `Preferences: User Settings` and set `apexPMD.pmdPath` to folder where pmd was unzipped in step 3
 
 ## Configuration
 
 ``` javascript
     // absolute path to where PMD was installed
-    "apexPMD.pmdPath": "/Users/johndoe/pmd",
+    // For example : /Users/johndoe/pmd on macOS
+    // C:\\Code\\pmd-bin-6.7.0 on windows
+    "apexPMD.pmdPath": "/Users/johndoe/pmd", // on Mac 
 
     // Set to false to use you own ruleset (set path)
     "apexPMD.useDefaultRuleset": true,
 
     // Path to ruleset xml file.  Must also set `useDefaultRuleset:false`.
-    // If relative - current workspace root dir is used as a starting point
-    "apexPMD.rulesetPath": "",
+    // If relative - current workspace root dir is used as a starting point.
+    // Path in this format for mac "apexPMD.rulesetPath": "/Users/johndoe/pmd"
+    // in this format for Windows "apexPMD.rulesetPath": "C:\\Code\\ruleset\\apex_default.xml"
+    "apexPMD.rulesetPath": "", // on Mac
```

Comment:
I don't quite understand what does `// on Mac` comment relate to since it's an empty string?
Please, remove this comment